lab-edward-smith/lib/readFiles.js

```diff
@@ -0,0 +1,15 @@
+const Promise  = require('promise');
+const fs       = require('fs');
+
+const readFiles = exports = module.exports = {};
+
+
+readFiles.readFile = function(file) {
```

Comment:
Your function should be able to read an array of files, rather than just individual ones.